shared/src/ledger/native_vp/masp.rs

```diff
@@ -178,6 +179,7 @@ where
         &self,
         transaction: &Transaction,
     ) -> Result<bool> {
+        println!("\nCHECKING VALID NOTE COMMITMENT\n");
```

Comment:
Are debug codes left in this file?

Reply:
removed in a1143ab7f88fa58cce305c6981dfdb6924887476